CHANGELOG.md

```diff
@@ -1,10 +1,40 @@
 # Releases
 
-## Next (unreleased)
+## v1.2.0 (May 14, 2019)
+
+### Enhancements
+
+* **Trace detail:** Limit the thickness of spans in the minimap ([@rubenvp8510](https://github.com/rubenvp8510) in [#372](https://github.com/jaegertracing/jaeger-ui/pull/372))
+
+* **UI find:** Scroll to first match on load or on press of new locate icon ([@everett980](https://github.com/everett980) in [#367](https://github.com/jaegertracing/jaeger-ui/pull/367))
```

Comment:
I suggest renaming "UI find" to "Search in trace"

Reply:
@yurishkuro we've been trying to separate the terms "Find" and "Search" as "find in loaded data" vs "request new data"
Also "in trace" isn't always the right label as there are multiple traces for a diff, and while DDG data is generated from traces, it isn't trace data.